pkgs/games/dwarf-fortress/df_permission/2024

```diff
@@ -0,0 +1,57 @@
+From: Tarn Adams <tarn.adams@gmail.com>
```

Comment:
same here

Reply:
same, this is email permission from the developer to allow us to package it in nixpkgs